src/lib/project.js

```diff
@@ -323,10 +340,67 @@ async function scaffoldNext() {
         ],
       },
     ];
-  }
+  },
 };`
   );
+  newNextConfig = newNextConfig.replace(
+    'reactStrictMode: true',
+    'reactStrictMode: false'
+  );
   fs.writeFileSync(path.join('ui', 'next.config.js'), newNextConfig);
+
+  const indexFileName = useTypescript === 'yes' ? 'index.tsx' : 'index.js';
+
+  fs.writeFileSync(
+    path.join('ui', 'pages', indexFileName),
+    newNextIndex,
+    'utf8'
+  );
+
+  // TODO: The command npm run build needs to be invoked in the smart contract directory root to generate
+  // the build that is imported into the UI
+
+  newNextConfig = newNextConfig.replace(
+    'reactStrictMode: true',
+    'reactStrictMode: false'
+  );
+  fs.writeFileSync(path.join('ui', 'next.config.js'), newNextConfig);
+
+  const tsconfig = `
```

Comment:
Ok for now. Future, we can consider if we want to create the `contracts` dir first and read this file from it (less to maintain).